docs/user-guide/faq.md

```diff
@@ -0,0 +1,67 @@
+# FAQ
+
+## How do I lint within my text editor?
+
+The stylelint community maintains a [handful of plugins](/docs/user-guide/complementary-tools.md#editor-plugins) for popular editors. Refer to their individual READMEs to get started.
+
+## How do I lint from the command line?
+
+First, install stylelint using the `-g` option:
+
+```shell
+npm install -g stylelint
+```
+
+Then refer to the [CLI usage examples](/docs/user-guide/cli.md) within the doc.
+
+The CLI can also be used from within [npm run scripts](http://blog.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/) if you'd rather use them over a task runner like `gulp`.
+
+## How do I use stylelint with gulp?
+
+Use [`gulp-stylelint`](https://github.com/olegskl/gulp-stylelint).
+
+## How do I lint using webpack
+
+Use [stylelint-webpack-plugin](https://github.com/vieron/stylelint-webpack-plugin).
+
+## How do I lint using other task runners like grunt?
+
+If there isn't a dedicated stylelint plugin for your task runner of choice, you can use stylelint as a PostCSS plugin and make use of PostCSS's [numerous](https://github.com/postcss/postcss#runners) task runner plugins.
+
+There are example of using the PostCSS plugin within the [docs](/docs/user-guide/postcss-plugin.md).
+
+## How do I lint SCSS, Less, or other non-standard syntax?
+
+stylelint can *parse* any the following non-standard syntaxes: SCSS, less and SugarSS. Refer to the [docs](/docs/user-guide/css-processors.md#parsing-non-standard-syntax) on how to configure stylelint to parse one of these syntaxes.
+
+## How do I automatically fix stylistic warnings?
+
+[stylefmt](https://github.com/morishitter/stylefmt) supports stylelint configuration files and can automatically a number of stylistic warnings.
+
+## Can I use stylelint as a PostCSS plugin?
+
+[Yes you can](/docs/user-guide/postcss-plugin.md), but it's not our recommended method as the CLI and Node API (used within the likes of `gulp-stylelint` and `webpack-plugin-stylelint` is more fully-featured.
+
+## Should I lint before or after processing my stylesheets through PostCSS plugins or other processors?
+
+We [recommend](/docs/user-guide/css-processors.md) linting your source files before any transformations.
+
+## How should I lint my CSS that follows a BEM-like methodology?
+
+You'll want to make use of the [`stylelint-selector-bem-pattern`](https://github.com/davidtheclark/stylelint-selector-bem-pattern) plugin.
+
```

Comment:
You can also take advantage of `selector-*` rules to ban certain categories of selectors (e.g. id selectors) and control specificity.